manifests/complete-postgres-manifest.yaml

```diff
@@ -8,9 +8,21 @@ spec:
   - name: date
     image: busybox
     command: [ "/bin/date" ]
+  sidecars:
```

Comment:
We already have a sidecar example in the manifest.
Also keep in mind, that many users start testing the operator with the provided manifests.
Therefore, dummy code should be in comments.

Reply:
In that case, complete-postgres-manifest needs more things commented out, like the replication slots. And init_containers are not sidecars as such, IMO.

Reply:
I was referring to [this](https://github.com/zalando/postgres-operator/blob/master/manifests/complete-postgres-manifest.yaml#L75).

Reply:
Oh, right, never mind me then. I seem to be slightly behind the times. :)

I'll drop the sidecar bit.